Automattic/jetpack
# Publicize Components: Fix the panel refactor

As part of #25993 we moved the PublicizePanel to the publicize-components package, but as part of that move the tweet storm component was incorrectly imported, and meant that the feature was disabled.

Also, we refactored how the URL for the connections management page was set, but stopped passing it as a prop to the form component.

#### Changes proposed in this Pull Request:

* This PR changes the TwitterOptions import, so that the component is correctly imported
* It passes the `enableTweetStorm` prop to the panel in Jetpack
* It passes the `connectionsAdminUrl` to the PublicizeForm component.

#### Other information:


#### Jetpack product discussion


p1666807305300219-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:
* Enable Publicize/Jetpack Social and connect twitter
* Create a test post
* With this PR you should see the twitter options to share the post as a thread.
* Without it, those options will be missing.

Additionally you can follow the instructions in #25803 to display the broken connection message and check that the connections URL is correctly displayed.

#### Before

<img width="283" alt="image" src="https://user-images.githubusercontent.com/96462/198110456-f7985cb2-3717-4f2e-8fe5-11613ad787bb.png">

#### After

<img width="283" alt="image" src="https://user-images.githubusercontent.com/96462/198110270-2b609b0e-2066-47c2-a027-fb00f4dcbf58.png">
